packages/components/src/FormatFile/FormatFile.css

```diff
@@ -6,6 +6,7 @@
   --file-card--border-color: var(--color-border);
   --file-card--accent-color: var(--color-grey);
   --file-card--base-padding: var(--space-small);
+  --border-basebackground: var(--color-surface--background);
```

Comment:
Why are we defining a custom property here that we're only using in one place, as a background color? I'm only seeing this used on line 32.

Reply:
```suggestion
```

Reply:
I agree with @chris-at-jobber. If it is needed I would like to see it prefixed with `--file-card...`